lisa/main.py

```diff
@@ -87,6 +87,14 @@ def initialize_runtime_folder(
     working_path: Optional[Path] = None,
     run_id: str = "",
 ) -> None:
+    global _runtime_root
+    
+    # Update _runtime_root if working_path is provided
+    if working_path and working_path.is_absolute():
+        _runtime_root = working_path / "runtime"
+    elif working_path:
+        _runtime_root = Path(working_path).absolute() / "runtime"
+    
     cache_path = _runtime_root.joinpath("cache")
```

Comment:
It should be fixed by set cache_path, after the working path is set. It doesn't need to change the _runtime_root. Make the change simpler.